mlflow/environment_variables.py

```diff
@@ -863,3 +863,24 @@ def get(self):
 #: Whether to disable telemetry collection in MLflow. If set to True, no telemetry
 #: data will be collected. (default: ``False``)
 MLFLOW_DISABLE_TELEMETRY = _BooleanEnvironmentVariable("MLFLOW_DISABLE_TELEMETRY", False)
+
+
+#: Maximum number of telemetry records to include in each batch.
+#: (default: ``10``)
+_MLFLOW_TELEMETRY_BATCH_SIZE = _EnvironmentVariable("_MLFLOW_TELEMETRY_BATCH_SIZE", int, 10)
+
+#: The time interval in seconds to wait before sending a batch of telemetry records.
+#: (default: ``30``)
+_MLFLOW_TELEMETRY_BATCH_TIME_INTERVAL = _EnvironmentVariable(
```

Comment:
q: what happens when a script is terminated before flushing the logs? Will we flush all remaining logs before terminated?